Inspect_agent/inspect_agent.py

```diff
@@ -58,11 +21,15 @@ def save_discrete(object):
     with open('answers.json', 'w') as f:
         json.dump(data, f)
 
-dataset = json_dataset("placeholder.json", record_to_sample)
-
 #SYSTEM_MESSAGE = read_file("prompt_basic.txt")
 SYSTEM_MESSAGE = read_file("prompt_blacklist.txt")
 
+@solver
+def shuffle_files():
+    async def solve(state: TaskState, generate: Generate):
+        await sandbox().exec(["python3", "shuffle_files_script.py", "data"])
```

Comment:
This runs the python script inside the Docker container. I wrapped it in a `solver` so that I could pass it to the init parameter below.

Reply:
This was the neatest way I could find of doing this with my limited knowledge of Inspect